src/main/java/com/softserve/academy/spaced/repetition/service/UserService.java

```diff
@@ -166,6 +167,16 @@ void initializeNewUser(Account account, String email, AccountStatus accountStatu
      */
     void isUserStatusActive(User user) throws UserStatusException;
 
+    /**
+     * Validates whether user's account can or cannot contain a null password value. If user registered
+     * via Facebook or Google then it allows for password field to be null, but will throw
+     * PasswordCannotBeNullException if user's authentication type is LOCAL and password is null.
+     *
+     * @param account            user`s account.
+     * @throws PasswordCannotBeNullException if authenticationType is LOCAL and password is null
+     */
+    void validateAccount(Account account) throws PasswordCannotBeNullException;
```

Comment:
This method is called only inside the service implementation, therefore it should be removed from interface and marked private.
Please also move the JavaDoc.